app/models/rules/majority_unsheltered.rb

```diff
@@ -0,0 +1,15 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+class Rules::MajorityUnsheltered < Rule
+  def clients_that_fit(scope, requirement, opportunity)
+    if Client.column_names.include?(:majority_sheltered.to_s)
+      scope.where(majority_sheltered: ! requirement.positive)
```

Comment:
Does this not need to ignore nil?

Reply:
I don't think so -- SQL NULL is neither true nor false.